Split/Network/HttpClient/HttpDataRequest.swift

```diff
@@ -106,18 +106,17 @@ class DefaultHttpDataRequest: BaseHttpRequest, HttpDataRequest {
         }
     }
 
-    func getResponse(errorSanitizer: @escaping (JSON, Int) -> HttpResult<JSON>,
+    func getResponse(errorHandler: @escaping (JSON, Int) -> HttpResult<JSON>,
                      completionHandler: @escaping (HttpDataResponse<JSON>) -> Void) -> Self {
 
         self.response(
                 queue: DispatchQueue(label: HttpQueue.default),
                 responseSerializer:
-            DefaultHttpDataRequest.responseSerializer(errorSanitizer: errorSanitizer)) { response in
+            DefaultHttpDataRequest.responseSerializer(errorSanitizer: errorHandler)) { response in
```

Comment:
Just a comment for consistency, but if we are renaming `errorSanitizer` for `errorHandler`, maybe we can do the same at `DefaultHttpDataRequest.responseSerializer` (if the semantic is OK and there is no conflict with other identifiers)